.github/workflows/azuresync.yml

```diff
@@ -5,7 +5,8 @@ on:
     types:
       [opened, edited, deleted, closed, reopened, labeled, unlabeled, assigned]
 
-concurrency: azuresync
+concurrency: 
+  group: azuresync-${{ github.event.issue.number }}
```

Comment:
No need for group. `concurrency: azuresync-${{ github.event.issue.number }}` does the same